java/dev/checkstyle/checkstyle.xml

```diff
@@ -223,14 +224,14 @@
         <module name="JavadocMethod">
             <property name="scope" value="public"/>
             <property name="allowMissingParamTags" value="true"/>
-            <property name="allowMissingThrowsTags" value="true"/>
+<!--            <property name="allowMissingThrowsTags" value="true"/>-->
```

Comment:
will remove these once CIs are verified.

Reply:
Is this ready or not ready? Please keep PRs in draft until they are ready.

Reply:
This should have been on draft. Sorry about the mistake.